### PR TITLE
add storybook setup

### PR DESCRIPTION
**Pull request by chmelevskij**
Currently it's hard to contribute to UI only changes, since it's all tangled up with the logic and firmware. Proposing to use storybook setup, which would allow to migrate and work on components in isolation from the FC and firmware.

to run
```
yarn
yarn storybook
```

open localhost:6006

Screenshot 2022-04-23 at 18 50 06

### CODE REVIEW

File: .storybook/main.js
Comment (haslinghuis) [inline .storybook/main.js:1]:
```suggestion
module.exports = {
```

File: .storybook/main.js
Comment (haslinghuis) [inline .storybook/main.js:15]:
```suggestion
}

```

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
